### PR TITLE
[Salvador Villalon] Add i18n pen size translation

### PR DESCRIPTION
**Pull request by salvillalon45**
## Overview
This change adds translation strings for the change pen size tool found in the top left above tools. 

We use the function fillIntTemplate found in utils/Template.js to help us generate templates with values in them

New templates in misc-templates.html have been added.

drawing-tools.html now does not have any static content for the pen-size-container section. Instead, it has comments saying where the context gets created from.

## Testing Done
In the picture, you can see that the Settings Option now has the TEST N in each text. Highlighted in red. This comes from en_us.json which means that the translation did go through.
We also see that we successfully generated the HTML content and we longer need to write HTML in drawing-tools.html

image (13)

## Follow Up
Going to work on Add New Frame Button

### CODE REVIEW

File: i18n/locales/en_us.json [hunk 1]
```diff
@@ -50,6 +50,8 @@
   "duplicateThisFrameFramesListTool" :"Duplicate this frame",
   "deleteThisFrameFramesListTool": "Delete this frame",
   "loadingPiskelMessage": "Loading Piskel...",
+  "penSizeChangeTitle": "Pen size TEST 1",
+  "penSizeChangeAt14816": "at 1, 4, 8, or 16 pixels TEST 2",
```
Comment (daynew) [inline i18n/locales/en_us.json]: Looks like you left some test data in here.

Comment (daynew) [inline i18n/locales/en_us.json]: The 14816 is kind of odd. Maybe `penSizeChangeAvailableSizes` or something like that?

File: i18n/locales/en_us.json [hunk 2]
```diff
@@ -80,5 +82,5 @@
   "gifExportSettingSectionUploadDesc": "Upload as an animated GIF to a public URL.",
   "gifExportSettingSectionUncheckToPlay": "Uncheck to play the animation only one time.",
   "gifExportSettingSectionBuildingAnimatedGif": "Building animated GIF ...",
-  "gifExportSettingSectionYourImageAvailableAt": "Your image is now available at :"
+  "gifExportSettingSectionYourImageAvailableAt": "Your image is now available at"
```
Comment (daynew) [inline i18n/locales/en_us.json]: This is from a different PR, right?

File: src/js/controller/PenSizeController.js [hunk 3]
```diff
@@ -24,4 +25,19 @@
     var size = pskl.app.penSizeService.getPenSize();
     this.sizePicker.setSize(size);
   };
+
+  ns.PenSizeController.prototype.createChangePenSize = function (i18n) {
+    var templateValues = {
+      title: i18n.penSizeChangeTitle() + '</br>' + i18n.penSizeChangeAt14816(),
```
Comment (daynew) [inline src/js/controller/PenSizeController.js]: I'm suprised the `<br>` tag can be used in the `title` attribute of an HTML element. I would expect only normal text. Would a newline character `\n` work as well?

**Review by daynew — CHANGES_REQUESTED** [review]

File: src/js/controller/PenSizeController.js (hunk 3 above)
Comment (salvillalon45) [reply src/js/controller/PenSizeController.js]: Going to leave as is

**Review by daynew — APPROVED** [review]

**Pull request merged**